tracer/test/test-applications/integrations/Samples.Elasticsearch/Properties/launchSettings.json

```diff
@@ -9,10 +9,10 @@
 
         "CORECLR_ENABLE_PROFILING": "1",
```

Comment:
I think the COR_PROFILER_PATH for all of these `launchSettings.json` files should be updated as well

Reply:
Yeah, I must have missed a find and replace somehow, thanks! 🤦‍♂️